source/documentation/administration_guide/topics/proc-adding-kubevirt-openshift-as-an-external-provider.adoc

```diff
@@ -34,9 +34,12 @@ NOTE: This capability is known as _OpenShift Virtualization_.
 
 . Click the name of new cluster you just created. This cluster name, *kubevirt* for example, is based on the name of the provider. This action opens the cluster details view.
 
-. Click the *Hosts* tab to verify that the {container-platform} status of the worker nodes is up.
+. Click the *Hosts* tab to verify that the status of the {container-platform} worker nodes is `up`.
 +
-NOTE: Even if they are running, the status of the master nodes is down because they cannot host virtual machines.
+[NOTE]
+====
+The status of the control plane nodes is `down`, even if they are running, because they cannot host virtual machines.
+====
 
 . Use menu:Compute[Virtual Machines] to deploy a virtual machine to the new cluster.
```

Comment:
Whoa! "Use" instead of "Click"? Someone either wasn't paying attention or didn't verify this in the web console. I won't hold up the merge for this but it's not good writing.